### DISCUSSION TITLE
Is there a graphical layout of a site's sub pages to reduce my confusion?

### DISCUSSION THREAD

**Discussion by ineuw** [community/community#58960] [Programming Help]
Is there a way that I can map the structure of the Autokey documentation site? I am lost in their documentation tree. I posted a Python code snippet yesterday on their site, and have no idea where it ended up. Thanks in advance.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ineuw ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
